cmd/internal/base/base.go

```diff
@@ -65,12 +65,15 @@ func (IDType) String() string {
 // InitCmd returns the init command for ent/c packages.
 func InitCmd() *cobra.Command {
 	var target string
+	var newSchemaTemplate string
```

Comment:
```suggestion
	var target, tmpl string
```

Keep variable names short please.